Dockerfile

```diff
@@ -1,23 +1,33 @@
-FROM python:3.9-slim as build
+FROM node:lts-buster as node-build
+WORKDIR /root
+COPY package.json .
+RUN npm --registry https://registry.npm.taobao.org install
+COPY . .
+RUN make assets-build
 
-RUN apt-get update && apt-get install -y g++ gcc libxslt-dev make libcurl4-openssl-dev build-essential
 
+FROM python:3.9-slim as build
+RUN apt-get update && apt-get install -y g++ gcc libxslt-dev make libcurl4-openssl-dev build-essential
 RUN apt-get install -y libssl-dev
+WORKDIR /root
 
-RUN mkdir -p /var/www/scylla
-WORKDIR /var/www/scylla
-
-RUN pip install scylla
+COPY --from=node-build /root/scylla/assets ./scylla/assets
+COPY requirements.txt .
+RUN pip install -r requirements.txt -i https://mirrors.aliyun.com/pypi/simple/
+RUN python -m playwright install
+COPY . .
+RUN python setup.py install
 
-FROM python:3.9-slim
+FROM python:3.9-slim as prod
```

Comment:
Why `as prod` is needed

Reply:
I added a  docker-compose file which is under testing, `prod` is used for specific target in docker-compose file's `build`

Reply:
oops, changes for testing are sending by pr 😭

Reply:
please just review b29969249ac4df1cc178aa019f33d23cd9b17d30 😭😭😭😭

Reply:
Can you remove this change?

Reply:
Yes, of course, give one night and I'll merge them into another pr.

Reply:
Sure! Let me know if your PR is ready. You could also mark this one as a draft PR if needed.